### PR TITLE
Make ColorModel and UIColor extension work for both platforms from one file each.

### PR DESCRIPTION
**Pull request by aranasaurus**
Small amount of refactoring required:
- Make offset a property on the color model which defaults to the current timezone's offset
- Make the ColorViewController on the phone update this property on willAppear using the offset saved in UserDefaults.

### CODE REVIEW

File: Chromatic/ColorModel.swift [hunk 1]
```diff
@@ -47,22 +50,19 @@ class ColorModel: NSObject {
             NSCalendarUnit.Minute
             ], fromDate: date)
         
-        self.didUpdate!(timeString: dateString as String, hex: hexString as String, color: color, hours: components.hour, minutes: components.minute)
+        updateBlock(timeString: dateString as String, hex: hexString as String, color: color, hours: components.hour, minutes: components.minute)
```
Comment (aranasaurus) [inline Chromatic/ColorModel.swift:53]: Sorry about the completely unrelated change... but I saw a `!` that was easy to get rid of and I couldn't help myself ;)

**Comment by alexpersian** [comment]
Looks good. Merging.
